python-sdk/nuscenes/map_expansion/map_api.py

```diff
@@ -126,6 +140,9 @@ def _load_layers(self) -> None:
         self.road_divider = self._load_layer('road_divider')
         self.lane_divider = self._load_layer('lane_divider')
         self.traffic_light = self._load_layer('traffic_light')
+        self.arcline_path_3 = self._load_arcline_path()
+        self.lane_connector = self._load_layer('lane_connector')
+        self.connectivity = self._load_lane_connectivity()
```

Comment:
Let's make sure to explain these in the map tutorial etc.:
https://www.nuscenes.org/map-tutorial

Reply:
Ok I will add in this PR?